### DISCUSSION TITLE
Plot multiple orbit segments on one graph - legend keeps repeating due to segment loop

### DISCUSSION THREAD

**Discussion by Bemtgenj** [groops-devs/groops#163] [Q&A]
Hello,

I want to plot orbit positions that are based on multiple segments / day (each segment having a name, day time stamp and a "start segment time stamp"). I am looping through the command output but every time i plot the next segment i get an additional legend on the right side. One way i fixed this problem was to use the first segment as a baseline and then run the other segments after that without a legend so the legend only is printed once. The problem with solution is that for a time series (1 plot / day  / month) is that the first segment might be different every day (so i cannot used this fixed first segment time stamp ) . 

Any good hint on how to solve this small issue.

I am not sure the question is clear. if needed i can upload one of the initial plots with the multiple legends

Thank you very much for the feedback and you help.

Cheers
Jean

**Comment by tmayerguerr** [comment]
Dear Jean,

In your loop you can define a variableLoopIndex='index'. Then you set a condition on the description in your layer with the expression 'index==0'

regards
Torsten

**Comment by Bemtgenj** [comment]
Dear Torsten

Txs for the quick reply, Let me try what you recommended above. If i have further questions i will let you know.

BR
Jean
